.gitmodules

```diff
@@ -1,3 +1,3 @@
 [submodule "cecil"]
-	path = cecil
+	path = src/cecil
```

Comment:
This should either be removed (as we discussed) in favour of binary dependency or called `external` to align with namings used for submodules

Reply:
I'll move it to `external/cecil`. We discussed a binary dependency, but I believe settled on keeping the submodule to enable us to get updates more quickly and also avoid redstributing it as a third-party dependency.